src/sweeper/backup.py

```diff
@@ -4,3 +4,19 @@
 backup.py
 A module that creates a gdb if it doesn't exist and inserts a feature class
 '''
+import arcpy 
+
+def backup_data(source_data, out_path):
+    # if the gdb does not exist, then create it
```

Comment:
please add a doc string that tells me what source_data and out_path are. [forklift](https://github.com/agrc/forklift/blob/master/src/forklift/core.py#L65-L74) has great examples

Reply:
thanks for the example